tests/functional/test-infra.sh

```diff
@@ -85,3 +108,12 @@ unset res
 res=$(set -eu -o pipefail; echo foo | expect 1 grepQuietInverse foo | wc -c)
 (( res == 0 ))
 unset res
+
+# `grepQuiet` does not allow newlines in its arguments, because grep quietly
+# treats them as multiple queries.
+( echo foo; echo bar; ) | expectStderr -101 grepQuiet $'foo\nbar' \
```

Comment:
```suggestion
{ echo foo; echo bar; } | expectStderr -101 grepQuiet $'foo\nbar' \
```
Don't need a whole subshell for this